### DISCUSSION TITLE
How to export module?

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#14116] [How to] [answered]
How to export Reservation module, so it can be installed again (refreshed) in separate yetiforce application?

**Comment by JohnMaverick** [comment, marked as answer]
OK got it:

require __DIR__ . '/include/main/WebUI.php';
$moduleName = 'Leads';
$moduleModel = \vtlib\Module::getInstance($moduleName);
$package = new vtlib\PackageExport();
$package->export($moduleModel, '', sprintf('%s-%s.zip', $moduleModel->name, $moduleModel->version), true);

**Comment by apcloic** [comment]
@JohnMaverick 
Another way is to change customized from "0" to "1" in vtiger_tab and then use GUI.
